docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/libraries/analytics/analytics.js

```diff
@@ -1,810 +1,358 @@
-import $ from 'jquery';
-import { registerCustomEventListener } from 'Core/libraries/customEventHandler';
-import { NCIAnalytics } from 'Core/libraries/analytics/nci-analytics-functions';
-import { AppMeasurementCustom } from 'Core/libraries/analytics/AppMeasurement.custom';
-
-// Make our 's' object accessible on the window.
-window.AppMeasurementCustom = AppMeasurementCustom;
-
-//utility functions
-// treeText
-// when clicking on an accordion button, get the accordion hierarchy and depth
-// ARGS
-// obj:$ collection - collection of accordion titles
-function treeText(obj) {
-    var str = "", depth = 0;
-    $(obj.get().reverse()).each(function (i, el) {
-        if (str == "") {
-            str = $(el).find("a:first").text();
-        } else {
-            str += ">" + $(el).find("a:first").text();
-        }
-        depth++;
-    });
-    return {string: str, depth: depth};
-}
+import $ from "jquery";
+import NCIAnalytics from "Core/libraries/analytics/atomic-analytics-functions/all-themes/index";
 
 // Set a name for the view port based on the current screen size
 function getWidthForAnalytics() {
-    var screen = '';
-    if (window.innerWidth) {
-        if (window.innerWidth > 1440) {
-            screen = "Extra wide";
-        }
-        else if (window.innerWidth > 1024) {
-            screen = "Desktop";
-        }
-        else if (window.innerWidth > 640) {
-            screen = "Tablet";
-        }
-        else {
-            screen = "Mobile";
-        }
+  var screen = "";
+  if (window.innerWidth) {
+    if (window.innerWidth > 1440) {
```

Comment:
what and why are these widths

Reply:
This is legacy code. 1440 is the old widescreen breakpoint, which we still have in many places in the legacy CSS and JS that reference this. Funny enough, I found that this resize event does not fire on Home/Landing because we do not use any of the legacy stuff on the "unsullied" NCIDS templates.